scripts/release-bit-verification.sh

```diff
@@ -17,6 +17,9 @@ shout() {
 }
 # Check SHASUM for all the binary files and there should be no difference
 
+#For erroring out in case of error
+set -e
```

Comment:
Can you move this as the first instruction (i.e., right after the shebang)? Just so it handles all subsequent commands below it.

Reply:
Also, because the script contains some commands that are piped, like `rpm2cpio ${1} | cpio -idmvD $WORKING_DIR` in L29, I would suggest setting `-o pipefail` too. Otherwise, the exit code of a piped command would be that of the last command in the pipe.

```suggestion
set -eo pipefail
```

Reply:
Also, because the script contains some commands that are piped, like `rpm2cpio ${1} | cpio -idmvD $WORKING_DIR` in L29, I would suggest setting `-o pipefail` too. Otherwise, the exit code of a piped command would be that of the last command in the pipe.

```suggestion
set -eo pipefail
```